lxdock/container.py

```diff
@@ -37,6 +40,9 @@ class Container:
     # The default image server that will be used to pull images in "pull" mode.
     _default_image_server = 'https://images.linuxcontainers.org'
 
+    # The default path for storing the command to execute during `lxdock shell`.
+    _guest_shell_script_file = '/.lxdock.d/shell_cmd.sh'
```

Comment:
This will probably bite us at some point (having a single hadcoded path for all shell commands). We might want to do the right thing right away and go with a `mktemp` approach.